.github/workflows/gradle.yml

```diff
@@ -247,6 +247,103 @@ jobs:
           SLACK_USERNAME: Buildbot
           SLACK_TITLE: "Build Success"
           SLACK_FOOTER: ""
+
+  ## Gradle Build (Platform)
+  # In case of self-hosted EC2 errors, remove this block.
+  start-platform-new-scheduler-acceptance-runner:
+    name: "Acceptance Tests (Docker w/ Scheduler v2)"
```

Comment:
This is for the Runner. it need to have Runner at the end  we need to put that on line 274

Reply:
Ah good catch, will fix